nashville/scooter-data/README.md

```diff
@@ -0,0 +1,40 @@
+# Nashville Shared Urban Mobility Device (SUMD) Parking Compliance
+
+Source:
+https://data.nashville.gov/browse?tags=scooter&utf8=%E2%9C%93
+
+CSV columns:
+- 'availability_duration': Duration of the parked device's availability (HH:MM:SS)
+- 'availability_start_date': Date of the parked device's availability (YYYY-MM-DD)
+- 'availability_start_time': Time of the parked device's availability (HH:MM:SS)
+- 'company_name': Name of scooter company
+- 'company_phone': SUMD vendor local customer service phone number
+- 'company_website': SUMD vendor website URL
+- 'gps_latitude': Latitude of the parked device
+- 'gps_longitude': Longitude of the parked device
+- 'real_time_fare': Real-time SUMD fare per unit distance
+- 'sumd_group': Group of device, e.g. "bicycle", "tricycle", "scooter", "hoverboard", "skateboard", "pedal car" or "other"
+- 'sumd_id': SUMD type and unique identifier for every device, determined by company
+- 'sumd_type': Locomotion type of device, e.g. "standard" or "powered"
+- 'availability_start_date_cst': availability_start_date converted to CST
+- 'availability_start_time_cst': availability_start_time converted to CST
+- 'extract_date': extract_date CST
+- 'extract_time': extract_time CST
```

Comment:
Honestly, it might be clearer to use UTC or at least a UTC offset because the time zone acronyms change throughout the year.

Reply:
Yes! I added in an additional set of columns for the extract date/time in UTC and relabeled the existing extract_date and time columns with '_cst' to ensure to make it clearer.